drake/util/Polynomial.h

```diff
@@ -10,9 +10,30 @@
 #include <stdexcept>
 #include "drake/drakePolynomial_export.h"
 
-// represents a scalar multi-variate polynomial
-// modeled after the msspoly in spotless
-
+/// A scalar multi-variate polynomial, modeled after the msspoly in spotless.
```

Comment:
I tested the doxygen build and it does work.  Also I believe that autobrief is only required if you want the first single-line sentence to be treated as the brief rather than the first separate comment block.